worker/include/RTC/TransportCongestionControlServer.hpp

```diff
@@ -54,10 +55,12 @@ namespace RTC
 		double GetPacketLoss() const;
 		void IncomingPacket(uint64_t nowMs, const RTC::RtpPacket* packet);
 		void SetMaxIncomingBitrate(uint32_t bitrate);
+		void FillAndSendTransportCcFeedback();
```

Comment:
This method must be private and hence moved below. And must also be changed in the cpp (please move it to the proper position to respect the order of the methods as defined here in the .hpp file).

Reply:
Mmmm, the problem here is that the test file calls this method directly, so it cannot be private, right?

Reply:
If so don't make it privsate, but please move `FillAndSendTransportCcFeedback()` declaration above `SetMaxIncomingBitrate()` to keep same order as in the cpp file.

Reply:
Pushed.